xdsl/parser/core.py

```diff
@@ -792,6 +797,32 @@ def _parse_op_result_list(self) -> list[tuple[Span, int]]:
     def parse_optional_attr_dict(self) -> dict[str, Attribute]:
         return self.parse_optional_dictionary_attr_dict()
 
+    def resolve_operands(
+        self, args: list[UnresolvedOperand], input_types: ArrayAttr[Attribute],
+        error_pos: Span
```

Comment:
```suggestion
        error_pos: Position
```
Since we anyway don't use the `end` part.